script/release

```diff
@@ -557,7 +557,7 @@ $release->_debug( "Will use HARNESS_OPTIONS '$ENV{HARNESS_OPTIONS}' during tests
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # Check that this is a version later than the one of CPAN
-unless( $opts{V} // $release->config->skip_cpan_version_check ) {
+unless( $opts{V} // $skip_dist || $release->config->skip_cpan_version_check ) {
```

Comment:
Version checking needs to make the dist too, but I don't think you'd want this check for what you are doing. Would `release -V -t` work?